integrated_channels/sap_success_factors/models.py

```diff
@@ -45,7 +45,8 @@ class SAPSuccessFactorsEnterpriseCustomerConfiguration(ConfigurationModel):
     The Enterprise specific configuration we need for integrating with SuccessFactors.
```

Comment:
I want to make sure I'm properly understanding why we're using a ConfigurationModel here - my understanding of that model is that it's designed to produce a single canonical configuration for the whole platform, with rollback features built in so that prior versions can be restored. Wouldn't we want to have any number of canonical models that are tied to different EnterpriseCustomers? We can use HistoricalRecords() to make sure the history of each model is saved.